apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -295,7 +295,8 @@ testBothFutureAndLegacyRoutes.describe("pro user", () => {
     await pageTwo.goto(initialUrl);
     await pageTwo.waitForURL(initialUrl);
 
-    await pageTwo.waitForSelector('[data-testid="event-type-link"]');
+    await pageTwo.waitForLoadState("networkidle");
+    await pageTwo.waitForSelector('[data-testid="event-type-link"]', { state: "visible" });
```

Comment:
I think state visible is the default. Isn't it?

Also [waitForSelect is discouraged](https://playwright.dev/docs/api/class-elementhandle#element-handle-wait-for-selector):

<img width="1014" alt="image" src="https://github.com/user-attachments/assets/43b69ca3-73a9-4296-8651-0299bbd0c349">

Reply:
Make sense thought, i will update